### PR TITLE
Update Cake-Issues Dependabot group

### PR DESCRIPTION
**Pull request by bdukes**
## Summary
Dependabot opened #6064 and #6065, which are both updates to Cake Issues packages, and should be grouped together. This PR updates the Cake-Issues Dependabot group to include the Cake.Frosting.Issues packages.

### CODE REVIEW

**Review by valadas — APPROVED** [review]

**Review by aspelt — APPROVED** [review]

**Comment by leonhardholz** [comment]
This failed to merge because @aspelt doesn't have write access. Can someone else from the DNN team approve?

**Comment by valadas** [comment]
It needs a second approval for a member in the @dnnsoftware/approvers group to merge.

**Review by david-poindexter — APPROVED** [review]
Thanks @bdukes

**Pull request merged**
